tests/typing_test.py

```diff
@@ -13,37 +13,14 @@
 import unittest
 from io import StringIO
 
-from nubia.internal.typing import command, argument
+from nubia.internal.typing import argument, command
 from nubia.internal.typing.argparse import add_command, find_command
 from nubia.internal.typing.builder import build_value
 
+from .parser_test_case import ContainedParser, ParseError, ParserTestCase
```

Comment:
I think you forgot to include `parser_test_case`, though I think it can be kept how it was as it's unrelated to the change being made mostly.